htmlParsing/rules.ts

```diff
@@ -0,0 +1,42 @@
+import { v4 as uuidv4 } from "uuid"
```

Comment:
node's built-in crypto package has a function to generate a v4 uuid. see https://nodejs.org/api/crypto.html#cryptorandomuuidoptions

Reply:
Sweet, didn't know :)